spec/lib/task_helpers/exports_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe TaskHelpers::Exports do
+  context 'with filename' do
```

Comment:
This should probably be a `describe ".safe_filename" do`

The convention in rspec is to `describe` the class/module under test then `describe` each method you are testing then use `it ".." do` blocks for each test case on a particular method.

Then `context` can be used for shared setup between `it` or `describe` blocks.